<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculatrice</class>
 <widget class="QMainWindow" name="Calculatrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>260</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculatrice</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>360</width>
     <height>260</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>350</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_M">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonClearAffich">
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 255);</string>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonClear">
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 255);</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonBack">
         <property name="maximumSize">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 255);</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="ButtonMemClear">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(157, 157, 157);
color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>MC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonMemRead">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(157, 157, 157);
color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>MR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonMemSet">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(157, 157, 157);
color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>MS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonMemPlus">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(157, 157, 157);
color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>M+</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="PushButton7">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PushButton4">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PushButton1">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PushButton0">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="PushButton8">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PushButton5">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PushButton2">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonSigne">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>SigneMoins</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="PushButton9">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PushButton6">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PushButton3">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonVirgule">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 0);</string>
             </property>
             <property name="text">
              <string>,</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButtonDiv">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(221, 221, 221);</string>
             </property>
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMulti">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(221, 221, 221);</string>
             </property>
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMoins">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(221, 221, 221);</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPlus">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(221, 221, 221);</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButtonSqrt">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(221, 221, 221);</string>
             </property>
             <property name="text">
              <string>sqrt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonInv">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(221, 221, 221);</string>
             </property>
             <property name="text">
              <string>1/x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPower2">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(221, 221, 221);</string>
             </property>
             <property name="text">
              <string>x²</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonEgale">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(221, 221, 221);</string>
             </property>
             <property name="text">
              <string>=</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>300</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>100</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
